### PR TITLE
Rearrange dashboard content

### PR DESCRIPTION
**Pull request by UrsMetz**
Now the description (what the dashboard is for and what is displayed) is moved above the actual content and is not placed in a panel any more. This seems more logical to me.

I also found some inconsistencies between the legacy and non-legacy version of the Pug templates and fixed them in separate commits. I'm not complete sure whether now the are equivalent because I didn't completely understand why they differ for some dynamic content.

### CODE REVIEW

**Comment by UrsMetz** [comment]
I just realized I forgot to delete the now obsolete panel :-(. I will push another commit when I've fixed that.

**Comment by NicoleRauch** [comment]
Look - I even created an issue for you :-)
https://github.com/softwerkskammer/Agora/issues/1129
You can assign yourself to it if you like.

**Comment by UrsMetz** [comment]
Cool :-), maybe I'm a bit blind but I can't find a button or anything like that to assign #1129 to me.

**Comment by NicoleRauch** [comment]
In the right hand side column there is this:

image

where I hope you can assign yourself. (I cannot assign you because you are not a member of the Softwerkskammer team but I hope you can do the assignment).

**Comment by NicoleRauch** [comment]
If that is not there, then this explains it ;-)

**Comment by UrsMetz** [comment]
Under assignees I only see "no one assigned" but there is no "assign yourself" and no cogwheel ("Zahnrädle") :-(.

**Comment by UrsMetz** [comment]
I removed the panel but left the panel with the upcoming activity at 6 bootstrap columns. Do you think that is okay. On mobile device it looks good (because the columns take the full width).
Maybe the text could be changed to first take about the upcoming activities and than about the user's groups because that's the order they are listed. Should I change that.

Should I squash some/all commits regarding the rearrangement of description text?

PS: the failing tests can't be my mistake ;-)

File: softwerkskammer/lib/dashboard/views/indexJson.pug [hunk 1]
```diff
@@ -10,15 +10,11 @@ block scripts
 block title
   | Dashboard
 block content
+  h1 #{t('dashboard.dashboard')}
+  !=t('dashboard.description', { postProcess: 'pug' })
   .row
```
Comment (leider) [inline softwerkskammer/lib/dashboard/views/indexJson.pug]: usually our headers look like that:
```javascript
      .page-header
        h2 #{t('activities.activities')}
```

**Comment by NicoleRauch** [comment]
No need to squash commits. I prefer to see how things evolved. But if you prefer, then pls feel free to squash.

**Comment by leider** [comment]
weird

**Comment by UrsMetz** [comment]
@leider done :-)
@NicoleRauch I also don't need them squashed. I was just asking because some projects a quite "squash-crazy"

**Comment by leider** [comment]
almost done. :) You missed the ".page-header"

**Comment by UrsMetz** [comment]
@leider just saw it; now done-done :-)

**Comment by leider** [comment]
Sorry, nope.
I'd prefer something like:
```
    h2 #{t('dashboard.dashboard')}
      small 
        !=t('dashboard.description', { postProcess: 'pug' })
  .row
```

The .row must be the same indentation as h2, otherwise the header will include the content :)

**Comment by UrsMetz** [comment]
@leider I see, but should `dashboard.description` really be inside the `h2` tag as a "sub heading"?

**Comment by leider** [comment]
Yes. Try it and have a look. Do you like what you see? - It will look similar to e.g. activity page or member page

**Comment by UrsMetz** [comment]
Okay, now I recognize the "corporate design" :-). As the description is a bit long I think on mobile it looks a bit "huge" (on desktop it is good IMHO):
2016-09-24-12-30-28_scrot

**Comment by leider** [comment]
It will scroll away. But we can also show a different text. What about: "Deine persönliche Softwerkskammer"? - You could adapt that in both language files. Maybe also change the "pug"-weird thing there, if you understand what I mean. :)

**Comment by UrsMetz** [comment]
I like the shorter text because that the activities and groups are shown should be obvious or could be made clear by changing "Kommende Aktivitäten" to "Kommende Aktivitäten Deiner Gruppen" and maybe a heading above the listings of the groups? Should I change the texts and structure like that?

Currently I don't know about the pug-weirdness think but maybe I will learn about it while making the changes :-).

**Comment by leider** [comment]
Yes, please! Go for it!

File: softwerkskammer/lib/dashboard/views/index.pug [hunk 2]
```diff
@@ -13,13 +13,15 @@ block title
 block content
   .page-header
     h2 #{t('dashboard.dashboard')}
-    !=t('dashboard.description', { postProcess: 'pug' })
-    .row
-      .col-sm-6
-        +my_activities
-    +my_groups
+      small
+        p #{t('dashboard.description')}
```
Comment (leider) [inline softwerkskammer/lib/dashboard/views/index.pug]: Das "p" ist zu viel - zumindest unnötig. Kannst Du mal schauen, wei der Unterschied ist zwischen mit und ohne?

File: locales/translation-de.json [hunk 3]
```diff
@@ -130,8 +130,10 @@
   "dashboard": {
     "dashboard": "Überblick",
     "description": [
-      "p Deine persönliche Softwerkskammer. Wir zeigen Dir hier Deine Gruppen und ihre Aktivitäten."
-    ]
+      "Deine persönliche Softwerkskammer"
+    ],
+    "upcoming_activities": "$t(activities.upcoming_activities) Deiner Gruppen",
+    "your_groups": "Deine Gruppen"
```
Comment (leider) [inline locales/translation-de.json]: Das funktioniert so? - Bin beeindruckt von pug. :)

File: softwerkskammer/lib/dashboard/views/index.pug [hunk 4]
```diff
@@ -13,13 +13,15 @@ block title
...
+  .row
+    .col-sm-6
+      +my_activities
+  +my_groups
 
 mixin my_groups
+  h3 #{t('dashboard.your_groups')}
```
Comment (leider) [inline softwerkskammer/lib/dashboard/views/index.pug]: machen wir jetzt "my" oder "your" ? :)

File: softwerkskammer/lib/dashboard/views/indexJson.pug [hunk 5]
```diff
@@ -12,11 +12,12 @@ block title
 block content
   .page-header
     h2 #{t('dashboard.dashboard')}
-    !=t('dashboard.description', { postProcess: 'pug' })
-    .row
-      .col-sm-6
-        +my_activities
-    +my_groups
+      small
+        p #{t('dashboard.description')}
```
Comment (leider) [inline softwerkskammer/lib/dashboard/views/indexJson.pug]: wie oben "p"

File: locales/translation-de.json (hunk 3 above)
Comment (UrsMetz) [inline locales/translation-de.json]: Ja, habe ich von einer anderen zusammengesetzten Übersetzung abgeschaut (sonst hätte ich das gar nicht ausprobiert) :-)

**Comment by leider** [comment]
Konnte es grad mal live testen. => Sieht SUPER aus!

File: softwerkskammer/lib/dashboard/views/index.pug (hunk 2 above)
Comment (UrsMetz) [inline softwerkskammer/lib/dashboard/views/index.pug]: Lassen wir das "p" dann doch stehen? Ohne "p" steht die description eben in der selben Zeile (und mit wenig bis gar keinem Abstand, warum das auf anderen Seiten, z. B. im Adminbereich habe ich noch nicht ganz verstanden, nicht so ist. Das ist auf Agora nicht ganz einheitlich: auf einer Acitivity-Seite steht die graue Unterüberschrift unterhalb der großen (aber mit einem "br" in die nächste Zeile gezwungen).

**Comment by leider** [comment]
Auf den anderen Seiten haben wir ein br. Aber das ist OK so. Ist ja jetzt drin!

**Comment by UrsMetz** [comment]
Okay, dann danke noch mal für die Geduld, wenn mein Kampf mit pug etwas länger gedauert hat :-).

**Comment by NicoleRauch** [comment]
Den Kampf fechte ich auch jedesmal wieder aufs Neue aus, keine Sorge ;-)

**Pull request merged**